app_dart/test/request_handlers/refresh_chromebot_status_test.dart

```diff
@@ -236,8 +277,9 @@ void main() {
     });
 
     test('update task status for non master branch', () async {
+      config.tabledataResourceApi = tabledataResourceApi;
```

Comment:
Nit: consider moving this line to setUp().

Reply:
Done.